src/main.cpp

```diff
@@ -332,8 +343,8 @@ int main(int argc, char** argv)
     soundManager->setMusicVolume(PreferencesManager::get("music_volume", "50").toFloat());
     soundManager->setMasterSoundVolume(PreferencesManager::get("sound_volume", "50").toFloat());
 
-    main_font = GuiTheme::getTheme("default")->getStyle("base")->states[0].font;
-    bold_font = GuiTheme::getTheme("default")->getStyle("bold")->states[0].font;
+    main_font = GuiTheme::getCurrentTheme()->getStyle("base")->states[0].font;
+    bold_font = GuiTheme::getCurrentTheme()->getStyle("bold")->states[0].font;
```

Comment:
While this is a bit of a hack to get a lot of legacy code to work, these main_font and bold_font globals are now not updated when you change a theme until you restart the game.

Reply:
Herrrr you're right I may update this when updating preferences.